AppCenterAuth/AppCenterAuth/MSAuth.m

```diff
@@ -262,6 +264,23 @@ - (void)signOut {
 
 #pragma mark - Private methods
 
+- (void)enableMSALLogging {
+  MSALGlobalConfig.loggerConfig.logLevel = MSALLogLevelVerbose;
+  [MSALGlobalConfig.loggerConfig setLogCallback:^(MSALLogLevel level, NSString *message, BOOL containsPII) {
```

Comment:
PII -> Pii?

Reply:
I'm using the same convention as MSAL defined.
We usually don't change this kind of things, do we?
[MSAL doc](https://github.com/AzureAD/microsoft-authentication-library-for-objc/wiki/Logging#logging-callback)

Reply:
camelCase is the first letter of every word upper case, I and I are initials for 2 separate words @achocron , thus to me it's PII.